src/components/MovieCard/MovieCard.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import './MovieCard.css'
+
+const MovieCard = props => {
```

Comment:
We can maybe consider changing this argument from props to a destructured object so we can just type {image} {title} etc.